prusti-tests/tests/verify/pass/arrays/merge.rs

```diff
@@ -0,0 +1,60 @@
+use prusti_contracts::*;
+
+fn main() {}
```

Comment:
I guess the postcondition of `merge` is not strong enough to verify e.g. `merge([1, 3, 5], [2, 4, 6])[2] == 3`, and we would need at least (multi)set conversions to do this?

What about a client test like this:
```suggestion
fn main() {
    let x = [1, 3, 5];
    let y = [2, 4, 6];
    let m = merge(x, y);
    assert!(m[1] <= m[4]);
}
```

Reply:
re `merge(..)[2] == 3`, i think so too. doesn't verify currently, and not specifying the output multiset seems like the obvious culprit.

added the suggested change in main.